net-misc/tiers/tiers-1.1-r1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="${PN}${PV}"
+DESCRIPTION="Random network topography generator"
+HOMEPAGE="https://www.isi.edu/nsnam/ns/ns-topogen.html#tiers"
+SRC_URI="https://www.isi.edu/nsnam/dist/topogen/${MY_P}.tar.gz
+	https://www.isi.edu/nsnam/dist/topogen/tiers2ns-lan.awk"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="mapm"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+RDEPEND="sci-visualization/gnuplot
```

Comment:
```suggestion
RDEPEND="
	sci-visualization/gnuplot
```

Reply:
Right. I'm looking at new ebuilds with my kde proj trained eye, and usually we do put lists of more-than-one dependencies into a neatly aligned block. @robbat2: what do you think?